assets/i18n/strings_pt-BR.i18n.json

```diff
@@ -17,7 +17,7 @@
         "error": "Erro",
         "example": "Exemplo",
         "files": "Arquivos",
-        "finished": "Terminado",
+        "finished": "Concluido",
```

Comment:
Change "Concluido" -> "Concluído" for grammatical accuracy.

Reply:
@TheGB0077 You can create a new PR. I will close this due to inactivity.